darts/ad/aggregators/aggregators.py

```diff
@@ -0,0 +1,300 @@
+"""
+Anomaly aggregators base classes
+"""
+
+# TODO:
+# - add customize aggregators
+# - add in trainable aggregators
+#     - log regression
+#     - decision tree
+# - create show_all_combined (info about correlation, and from what path did
+#   the anomaly alarm came from)
+
+from abc import ABC, abstractmethod
+from typing import Any, Sequence, Union
+
+import numpy as np
+
+from darts import TimeSeries
+from darts.ad.utils import _to_list, eval_accuracy_from_binary_prediction
```

Comment:
about _to_list() , there already exists in Darts a function called series2seq() in darts.utils.utils.py, might be worth reusing ?